src/main/java/metrics_influxdb/InfluxdbHttp.java

```diff
@@ -101,6 +101,26 @@ public InfluxdbHttp(String host, int port, String path, String database, String
 		);
 	}
 
+	/***
+	 *
+	 * @param version the Influxdb version, currently builds a different url for 0.9 versions, else defaults to the earlier url
+	 * @throws Exception
```

Comment:
No,  keep throw Exception to be free to throw any kind of exception (in future without breaking api)